Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -610,6 +610,13 @@ private class TopSitesDataSource: NSObject, UICollectionViewDataSource {
         cell.accessibilityLabel = cell.textLabel.text
         cell.removeButton.hidden = !editing
 
+        let removeButtonAccessibilityLabel =
+            String(format:
+                NSLocalizedString("TopSites.RemovePage.Button",
```

Comment:
Why not define this in `Shared/Strings.swift`?

I noticed we've added a number of other new strings with IDs that also weren't put in `Strings.swift`. Added this as a topic for tomorrow's meeting.